src/components/organisms/SettingsManager/Settings.tsx

```diff
@@ -60,6 +63,8 @@ export const Settings = ({
 
   const resourceRefsProcessingOptions = useAppSelector(state => state.main.resourceRefsProcessingOptions);
   const uiState = useAppSelector(state => state.ui);
+  const appConfig = useAppSelector(state => state.config);
+  const fileMap = useAppSelector(state => state.main.fileMap);
```

Comment:
here I wouldn't select the entire `appConfig` and `fileMap` as it will re-render the component on any changes to their propertis, it's better to select directly only the state that you need

Reply:
Done